code/cnn_1D_segm/train_fcn1D.py

```diff
@@ -0,0 +1,364 @@
+
+# coding: utf-8
+
+#!/usr/bin/env python2
```

Comment:
Oh, I didn't know. If time permits it, I can try but I doubt I will have time today..